data_model/src/transaction.rs

```diff
@@ -342,34 +350,53 @@ impl Transaction {
         self.payload.nonce = Some(nonce);
         self
     }
+}
 
-    /// Sign transaction with the provided key pair.
-    ///
-    /// # Errors
-    ///
-    /// Fails if signature creation fails
-    #[cfg(feature = "std")]
-    pub fn sign(
-        mut self,
-        key_pair: iroha_crypto::KeyPair,
-    ) -> Result<Transaction, iroha_crypto::Error> {
-        let signature = SignatureOf::new(key_pair, &self.payload)?;
-        self.signatures.insert(signature);
-
-        Ok(Self {
-            payload: self.payload,
-            signatures: self.signatures,
-        })
-    }
+/// Structure represents transaction in non-trusted form.
+///
+/// `Iroha` and its' clients use [`SignedTransaction`] to send transactions
+/// via network.  Direct usage in business logic is strongly
+/// prohibited. Before any interactions `accept`.
```

Comment:
I don't get it with `accept`. What do you mean?

Reply:
It's not my comment, but i suppose that `iroha` don't trust `SignedTransaction` before verifying signature and convert transaction into `AcceptedTrasaction`.

Reply:
I think this comment should be rephrased